internal/providersdkv2/resource_vault_cluster_test.go

```diff
@@ -321,7 +334,7 @@ func updatePublicProxyObservabilityAndMVU(t *testing.T, in *inputT) resource.Tes
 	}
 }
 
-// This step verifies the successful update of "tier", "public_endpoint", "proxy_endpoint" and removal of "metrics" and "audit_log"
+// This step verifies the successful update of "tier", "public_endpoint", "proxy_endpoint", "ip_allowlist" and removal of "metrics" and "audit_log"
 func updateTierPublicProxyAndRemoveObservability(t *testing.T, in *inputT) resource.TestStep {
```

Comment:
Same nit, maybe we can change it from `updateTierPublicProxyAndRemoveObservability` to `updateTierNetworkyAndRemoveObservability`where `Network` captures all three of public, proxy, and IP allowlist

Reply:
Renamed it!